connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerTaskTest.java

```diff
@@ -56,12 +53,11 @@ public class WorkerTaskTest {
     }
     private static final TaskConfig TASK_CONFIG = new TaskConfig(TASK_PROPS);
 
-    private ConnectMetrics metrics;
     @Mock private TaskStatus.Listener statusListener;
     @Mock private ClassLoader loader;
-    RetryWithToleranceOperator retryWithToleranceOperator;
-    @Mock
-    StatusBackingStore statusBackingStore;
+    @Mock StatusBackingStore statusBackingStore;
```

Comment:
Nit: for consistency's sake, this should be private:
```suggestion
    @Mock private StatusBackingStore statusBackingStore;
```